### PR TITLE
Support Web server for fuse process

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes are proposed in this pull request?

Support FUSE Web server, leverage this, we can get stacks, metrics restfulapi.

### Why are the changes needed?

We want to monitor FUSE server by prometheus, and get the thread stacks of FUSE in the web browser.

### Does this PR introduce any user facing changes?

Bring webserver to FUSE server.

If you want open fuse web server, set `alluxio.fuse.web.enabled=true` is necessary.

### CODE REVIEW

**Comment by maobaolong** [comment]
@LuQQiu Please take a look at this PR, thank you.

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java [hunk 1]
```diff
@@ -119,6 +121,14 @@ public static void main(String[] args) {
     }
     CommonUtils.PROCESS_TYPE.set(CommonUtils.ProcessType.CLIENT);
     MetricsSystem.startSinks(conf.get(PropertyKey.METRICS_CONF_FILE));
+    if (conf.getBoolean(PropertyKey.FUSE_WEB_ENABLED)) {
+      FuseWebServer mWebServer = new FuseWebServer(
```
Comment (LuQQiu) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java:125]: local variable doesn't need to be `mName`
change to `webServer`?

**Review by LuQQiu — COMMENTED** [review]
@tieujason330 Can you help double check is it the correct way to add a web server for Fuse process?

This PR is added to enable fuse process to expose metrics like what we did in master/worker
https://<fuse_process_host>:<fuse_web_port>/metrics/json or /metrics/prometheus/

**Comment by tieujason330** [comment]
> @tieujason330 Can you help double check is it the correct way to add a web server for Fuse process?
> 
> This PR is added to enable fuse process to expose metrics like what we did in master/worker
> https://<fuse_process_host>:<fuse_web_port>/metrics/json or /metrics/prometheus/

Looks good to me, then fuse-specific configurations will go into `FuseWebServer`

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java (hunk 1 above)
Comment (maobaolong) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java]: Sure, thanks for remind me.

**Comment by maobaolong** [comment]
@LuQQiu @tieujason330 Thanks for the previous review, PTAL.

**Comment by LuQQiu** [comment]
@apc999 PTAL, this change is likely to increase the alluxio tarball size since fuse depends on common now, is that overhead acceptable?

**Review by LuQQiu — APPROVED** [review]
LGTM! @maobaolong in the future, we will prefer to modulize worker, so that we can easily turn on/off worker, on/off fuse when needed

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**